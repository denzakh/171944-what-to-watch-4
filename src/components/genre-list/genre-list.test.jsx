import React from "react";
import renderer from "react-test-renderer";
import GenreList from "./genre-list.jsx";
import allGenreList from "../../mocks/genre-list.js";

describe(`Main Snapshot`, () => {
  it(`Should GenresList render correctly`, () => {
    const tree = renderer
      .create(
          <GenreList
            cactiveItem={`All genres`}
            allGenreList={allGenreList}
            setActiveItem={()=>{}}
          />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
