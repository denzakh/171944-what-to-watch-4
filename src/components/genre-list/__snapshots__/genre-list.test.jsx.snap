// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Snapshot Should GenresList render correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      All genres
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Action
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Adventure
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Comedy
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Crime
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Drama
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Mystery
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Sci-Fi
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Thriller
    </a>
  </li>
</ul>
`;
